packages/sveltekit/src/vite/sourceMaps.ts

```diff
@@ -60,14 +73,50 @@ export async function makeCustomSentryVitePlugins(options?: CustomSentryVitePlug
     },
   };
 
+  const filesToDeleteGlob = './.*/**/*.map';
+
+  if (sourceMapSetting.updatedSourceMapSetting === undefined) {
+    const configFile = await loadConfigFromFile({ command: 'build', mode: 'production' });
+
+    if (configFile) {
+      sourceMapSetting = getUpdatedSourceMapSetting(configFile.config);
+    } else {
+      if (options?.debug) {
+        consoleSandbox(() => {
+          // eslint-disable-next-line no-console
+          console.warn(
+            '[Source Maps Plugin] Could not load Vite config with Vite "production" mode. This is needed for Sentry to automatically update source map settings.',
+          );
+        });
+      }
+    }
+
+    if (options?.debug && sourceMapSetting.previousSourceMapSetting === 'unset') {
+      consoleSandbox(() => {
+        // eslint-disable-next-line no-console
+        console.warn(
+          `[Source Maps Plugin] Automatically setting \`sourceMapsUploadOptions.sourcemaps.filesToDeleteAfterUpload: ["${filesToDeleteGlob}"]\` to delete generated source maps after they were uploaded to Sentry.`,
```

Comment:
l: from user perspective, I think being high-level is easier to understand

```suggestion
            '[Sentry] Could not load Vite config with Vite "production" mode. This is needed for Sentry to automatically update source map settings.',
          );
        });
      }
    }

    if (options?.debug && sourceMapSetting.previousSourceMapSetting === 'unset') {
      consoleSandbox(() => {
        // eslint-disable-next-line no-console
        console.warn(
          `[Sentry] Automatically setting \`sourceMapsUploadOptions.sourcemaps.filesToDeleteAfterUpload: ["${filesToDeleteGlob}"]\` to delete generated source maps after they were uploaded to Sentry.`,
```